### ISSUE TITLE
JS: change how scroll-padding is evaluated

### ISSUE THREAD

**Issue by Megafry** [verbb/formie#1758]
### Describe the bug

Hi

The function `scrollToForm()` only checks for scroll-padding set as inline style.
The code should use `getComputedStyle()` to get the computed value.

fix:
```js
const extraPadding = parseInt(getComputedStyle(document.documentElement).scrollPaddingTop) || 0;
const extraMargin = parseInt(getComputedStyle(document.documentElement).scrollMarginTop) || 0;
```

https://github.com/verbb/formie/blob/craft-4/src/web/assets/frontend/src/js/formie-form-theme.js#L893-L894

> The value returned by `getComputedStyle` can be `auto` or any value in px

### Steps to reproduce

1. Add this style: `html { scroll-padding-top: 6.25rem }`
2. Create a new form with 2 pages.
3. Enable "Scroll To Top".
4. Switch between the pages to trigger the `scrollToForm()` function.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.7.3

### Plugin version

2.1.5

**Comment by engram-design** [comment]
Totally agree. Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.6"`.

**Comment by engram-design** [comment]
Fixed in [2.1.7](https://github.com/verbb/formie/releases/tag/2.1.7)
